src/chrome/content/rules/Geekzone.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="geekzone.co.nz (partial cdn)">
+	<target host="cdn.geekzone.co.nz" />
+	<target host="images.geekzone.co.nz" />

```

Comment:
See also:
geekzone.co.nz
www.geekzone.co.nz
feeds.geekzone.co.nz
live.geekzone.co.nz
m.geekzone.co.nz
price.geekzone.co.nz

You should list the issues in a comment block on the top if they are not working with HTTPS.